src/styles/utils/_mixins.scss

```
@@ -0,0 +1,11 @@
+@mixin field($size) {
+  width: $size;
+  height: $size;
+  box-sizing: border-box;
+  border: $field-border solid black;
+  background-color: #eee;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition-duration: 0.5s;
+}
```

This mixin is unnecessary here. All of these properties should be inside of __field in a regular way.

If you really want to use a mixin here, I think this would make more sense:
```suggestion
@mixin dimensions($size) {
  width: $size;
  height: $size;
}
```